src/plotman/job.py

```diff
@@ -370,7 +370,9 @@ def get_temp_files(self):
         # Prevent duplicate file paths by using set.
         temp_files = set([])
         for f in self.proc.open_files():
-            if self.tmpdir in f.path or self.tmp2dir in f.path or self.dstdir in f.path:
+            if self.tmpdir in f.path or self.dstdir in f.path:
+                temp_files.add(f.path)
+            if self.tmp2dir is not None and self.tmp2dir in f.path:
```

Comment:
What do you think of this?

```suggestion
            if any(
                dir in f.path
                for dir in [self.tmpdir, self.dstdir, self.tmp2dir]
                if dir is not None
            ):
```